lib/etl/load/create_new_resource_record.rb

```diff
@@ -2,9 +2,9 @@ class CreateNewResourceRecord
   def write(attributes)
     record = Resource.create!(attributes)
 
-    ap "Title: #{record.title}"
-    ap 'Metadata: '
-    ap record.metadata
-    ap "Content: #{record.content.truncate(100)}"
+    ap "Title: #{record.title}" # rubocop:disable Rails/Output
```

Comment:
What is Rubocop complaining about here? Any way to change it in the configuration? If we can't, maybe we could add a section to the Readme about the Rubocop offenses that should be disabled with `# rubocop:disable Rails/Output`.